package.json

```diff
@@ -22,6 +22,7 @@
   "author": "Wei <wei@liker.land>",
```

Comment:
You seems to have used a node version that causes package lock file to upgrade to v2
Please state your node.js version in package.json using `engine` field